patterns/1-initial/principles.md

```diff
@@ -0,0 +1,152 @@
+though with people lacking an open source background. As a remedy the most
+important principles of InnerSource get published widely.
+
+## Problem
+
+The organisation is trying to roll out InnerSource at a larger scale. The
+initiative started among open source enthusiasts. The goal is now to get buy-in
+from people that are lacking open source experience. For that audience the typical
+slogan of "applying open source best practices" is no longer sufficient to
+transport the message of what InnerSource is, which problems it solves and which
+tools it uses for solving these issues. As a result InnerSource adoption in the
+organisation slows down. Teams develop diverging ideas of what the goals of InnerSource
+is about and how to best implement it leading to confusion when contributors
+are starting to cross team boundaries. 
+
+## Story
+
+Early experiments in an organisation have shown that open source collaboration
+best practices can be beneficial. The goal now is to move the initiative to
+teams and individuals lacking a deep background in open source.
+
+The goal now is to clearly communicate the goals of the InnerSource initiative
+as well as a clear path towards achieving these goals.
+
+## Forces
+
+* Teams have trouble communicating exactly what the important aspects of
+InnerSource are.
+
+* People lacking open source experience fail to understand what it means to bring
+open source best practices into the organisation.
+
+* On a daily basis teams trying to follow InnerSource best practices have a hard
+time deciding if what they are doing is inline with general InnerSource values.
+
+## Solution
+
+Those that should adopt InnerSource need to understand two perspectives:
+
+### Why does the organisation want to adopt InnerSource?
+
+In the past InnerSource has proven to be successful to solve several issues
+commonly found in organisations. Instead of going for generalizations, try to
+exactly identify the solutions that match the challenges of your organisation -
+preferably with those affected by the change you want to see.
+
+Some challenges that others have addressed by following InnerSource
+best practices:
+
+* Reduce development silos caused by excessive ownership culture.
+* Increase innovation speed by reducing time spent solving similar issues by
+  fostering healthy code reuse.
+* Increase development speed by better cross team collaboration.
+* Solve project/ team dependencies beyond "wait it out" and "build workarounds",
+  thereby reducing engineering bottlenecks.
+* Increase quality.
+* Increase employee happiness.
+* To increase success of new hires.
+* To build actionable documentation.
+
+### Provide clear InnerSource principles
+
+Once teams understand which problems InnerSource will help them address, the next
+step is to explain which principles help address these challenges.
+
+Based on basic open source development principles the following guidelines
+have been proven successful:
```

Comment:
```suggestion
## Solution

Those driving the InnerSource initiative in the organisation need to
provide clarity around two areas:

### Why does the organisation want to adopt InnerSource?

In the past InnerSource has proven to be successful to solve several issues
commonly found in organisations. 

However which organizational challenges does your organization hope to improve
upon using InnerSource?

Instead of going for generalizations, try to exactly identify the solutions
that match the challenges of your organisation - preferably with those affected
by the change you want to see.

Some challenges that others have addressed by following InnerSource
best practices:

* Reduce development silos caused by excessive ownership culture.
* Increase innovation speed by reducing time spent solving similar issues by
  fostering healthy code reuse.
* Increase development speed by better cross-team collaboration.
* Solve project/ team dependencies beyond "wait it out" and "build workarounds",
  thereby reducing engineering bottlenecks.
* Increase quality.
* Increase employee happiness.
* To increase success of new hires.
* To build actionable documentation.

### Which InnerSource principles will help to address these challenges?

Once teams understand which problems InnerSource will help them address, the
next step is to explain which principles help address these challenges.

Based on basic open source development principles the following guidelines
have been proven successful:
```

Reply:
@MaineC this is my best effort to change the Solution section so that both points speak to the same "persona".

Maybe not very easy to review so here a list of my changes:

- changed the intro sentence of the Solution section back to what it was
- added some more context in the intro to "Why does the organisation want to adopt InnerSource?"
- changed the headline of the 2nd point to "Which InnerSource principles will help to address these challenges?", to make this more similar to the wording of the 1st point

Reply:
I like the way you changed the perspective. One thing I would add is information on who people driving the InnerSource initiative need to provide this clarity to.

Reply:
I changed the intro sentence to
> Those driving the InnerSource initiative in the organisation need to help the teams and individuals that are lacking a deep background in open source, and therefore have a less intuitive understanding of InnerSource.

Does that help?